conf/messages.en

```diff
@@ -2432,6 +2432,19 @@ companyNameOfSharesBuyer.error.required = Enter the buyer’s company name
 companyNameOfSharesBuyer.error.invalid = Buyer’s company name must only include letters a to z, numbers 0 to 9, apostrophes, ‘atʼ signs (@), forward slashes, full stops, hyphens, speech marks and spaces
 companyNameOfSharesBuyer.error.tooLong = Buyer’s company name must be 160 characters or fewer
 
+companyBuyerCrn.title = Do you know the buyer’s Company Registration Number (CRN)?
```

Comment:
This section looks to be duplicated. Double check please if all the content are the same, and if so this section can be removed.